examples/dt/dcn/service-values.yaml

```diff
@@ -0,0 +1,261 @@
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  preserveJobs: false
+  cinderAPI:
+    replicas: 3
+    customServiceConfig: |
+      [DEFAULT]
+        default_availability_zone = ceph
```

Comment:
Let's change this to az0 to align it with the name of compute nova AZ

Reply:
this has been addressed.